src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json

```diff
@@ -11,6 +11,7 @@
 
   // The variables are evaluated the first time they are accessed, and reused across steps
   "variables": {
+    "adminUrlPrefix": "[js: configuration('OrchardCore_Admin:AdminUrlPrefix', 'Admin']",
```

Comment:
```suggestion
    "adminUrlPrefix": "[js: configuration('OrchardCore_Admin:AdminUrlPrefix', 'Admin')]",
```

Reply:
Please can you try this locally, and validate that the menu items have picked up both the default, and a custom one.

Nothing in our tests will validate that this has been successful.

Reply:
Sure, I just facing an npm issue, I will review this PR today